crates/storage/db/src/metrics.rs

```diff
@@ -129,6 +148,30 @@ impl TransactionMetrics {
         if let Some(close_duration) = close_duration {
             metrics.close_duration_seconds.record(close_duration)
         }
+
+        if let Some(commit_latency) = commit_latency {
+            metrics
+                .commit_preparation_duration_seconds
+                .record(Duration::from_nanos(commit_latency.preparation()));
+            metrics
+                .commit_gc_wallclock_duration_seconds
+                .record(Duration::from_nanos(commit_latency.gc_wallclock()));
+            metrics
+                .commit_audit_duration_seconds
+                .record(Duration::from_nanos(commit_latency.audit()));
+            metrics
+                .commit_write_duration_seconds
+                .record(Duration::from_nanos(commit_latency.write()));
+            metrics
+                .commit_sync_duration_seconds
+                .record(Duration::from_nanos(commit_latency.ending()));
+            metrics
+                .commit_whole_duration_seconds
+                .record(Duration::from_nanos(commit_latency.whole()));
+            metrics
+                .commit_gc_cputime_duration_seconds
+                .record(Duration::from_nanos(commit_latency.gc_cputime()));
```

Comment:
I think returning the `Duration` from `commit_latency` methods would be better, so one doesn't have to guess what does this `u64` mean: nanoseconds, milliseconds, seconds, etc.